### DISCUSSION TITLE
Popper v3

### DISCUSSION THREAD

**Discussion by benmccann** [floating-ui/floating-ui#1712] [Q&A]
Any idea when Popper v3 might be released? Could a release be pushed to npm with the `next` tag?

**Comment by atomiks** [comment]
There is no Popper v3. Floating UI is sort of the next major but it's a lot different.

**Reply by benmccann** [reply]
Hmm. Would it be possible to cut a Popper v3? [Popper is still used by Bootstrap](https://github.com/floating-ui/floating-ui/issues/1718) and it causes a lot of problems that Popper v2 doesn't have an `exports` map. It did briefly have one, but the change was reverted as being a breaking change. I wonder if it'd be possible to merge something like https://github.com/floating-ui/floating-ui/pull/1526 as part of a new major release?

**Comment by sandstrom** [comment]
@atomiks Will there be an upgrade guide for Popper -> Floating UI?

**Reply by atomiks** [reply]
The upgrade guide is on the readme

**Reply by sandstrom** [reply]
Sorry, I'm blind! 🙈  I really did look for it…

If it's easy, maybe add a link in the docs website left-side navigation (I looked in those docs too, couldn't find it).

But mostly my fault, I agree it's clearly stated in the README.
